so.notion.Notion.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>so.notion.Notion</id>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-audio">intense</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
+  <developer_name>Notion Labs, Inc.</developer_name>
+  <name>Notion</name>
+  <summary>All-in-one workspace</summary>
+  <description>
+    <p>Notion is an application that provides components such as databases, kanban
+    boards, wikis, calendars and reminders.</p>
+    <p>Users can connect these components to
+    create their own systems for knowledge management, note taking, data management,
+    project management, among others.</p>
```

Comment:
It needs a disclaimer like https://github.com/flathub/com.microsoft.Teams/blob/master/com.microsoft.Teams.metainfo.xml#L16.